crates/bevy_audio/src/audio.rs

```diff
@@ -238,7 +238,7 @@ pub type AudioBundle = AudioSourceBundle<AudioSource>;
 ///
 /// When Bevy begins the audio playback, an [`AudioSink`][crate::AudioSink] component will be
 /// added to the entity. You can use that component to control the audio settings during playback.
-#[derive(Bundle)]
+#[derive(Bundle, Clone)]
```

Comment:
This should probably be manually implemented, right? I think this forces the generic bounds to impl `Clone` as well which isn't a necessary restriction since `Handle<T>` is `Clone` regardless of `T`